start_torq_demo.bat

```diff
@@ -27,9 +27,9 @@ start "discovery" q torq.q -load code/processes/discovery.q -proctype discovery
 timeout 2
 
 REM launch the tickerplant, rdb, hdb
-start "tickerplant" q torq.q -load code/processes/tickerplant.q -schemafile database -tplogdir %KDBTPLOG% -proctype tickerplant -procname tickerplant1 -U appconfig/passwords/accesslist.txt -localtime 
+start "segmentedtickerplant" q torq.q -load code/processes/segmentedtickerplant.q -schemafile database.q -tplogdir %KDBTPLOG% -proctype segmentedtickerplant -procname stp1 -U appconfig/passwords/accesslist.txt -localtime
 start "rdb" q torq.q -load code/processes/rdb.q -proctype rdb -procname rdb1 -U appconfig/passwords/accesslist.txt -localtime -g 1 -T 180
-start "chainedtp" q torq.q -load code/processes/chainedtp.q -proctype chainedtp -procname chainedtp1 -U appconfig/passwords/accesslist.txt -localtime
+start "segmentedchainedtickerplant" q torq.q -load code/processes/segmentedtickerplannt.q -proctype segmentedchainedtickerplant -procname sctp1 -U appconfig/passwords/accesslist.txt -localtime -parentproctype segmentedtickerplant
```

Comment:
`segmentedchainedtp`? At least it's marginally shorter and along the same lines as the old name.

Reply:
done